WorkloadManagementSystem/scripts/dirac-wms-get-wn-parameters.py

```diff
@@ -38,15 +38,25 @@ def setQueue( args ):
 Script.parseCommandLine( ignoreErrors = True )
 
 grid = Site.split( '.' )[0]
+gLogger.info("Getting numberOfProcessors from MJF")
 numberOfProcessor = JobParameters.getProcessorFromMJF()
 if not numberOfProcessor:
-  numberOfProcessor = gConfig.getValue( '/Resources/Sites/%s/%s/CEs/%s/Queues/%s/NumberOfProcessors' % ( grid, Site, ceName, Queue ) )
+  gLogger.info("NumberOfProcessors could not be found in MJF, trying from CS (queue definition)")
+  numberOfProcessor = gConfig.getValue( '/Resources/Sites/%s/%s/CEs/%s/Queues/%s/NumberOfProcessors' % (grid,
+                                                                                                        Site,
+                                                                                                        ceName,
+                                                                                                        Queue))
   if not numberOfProcessor:
+    gLogger.info("NumberOfProcessors could not be found in CS queue definition,\
+                  trying from /Resources/Sites/%s/NumberOfProcessors" % grid)
```

Comment:
Do you mean 2 arguments here or just line concatenation ? If the second, this should rather be:

  gLogger.info("NumberOfProcessors could not be found in CS queue definition, " \
                       "trying from /Resources/Sites/%s/NumberOfProcessors" % grid)

Otherwise there will be a gap between 'definition,' and 'trying'

Reply:
fixed.